facilitator/src/transport/s3.rs

```diff
@@ -192,13 +195,33 @@ impl Transport for S3Transport {
         info!("get {}/{} as {:?}", self.path, key, self.iam_role);
         let mut runtime = basic_runtime()?;
         let client = (self.client_provider)(&self.path.region, self.iam_role.clone())?;
-        let get_output = runtime
-            .block_on(client.get_object(GetObjectRequest {
+
+        let mut retries = 0;
+        let get_output = loop {
+            match runtime.block_on(client.get_object(GetObjectRequest {
```

Comment:
Style-wise, I think it's really valuale to avoid multi-line inputs to match statements. I recommend assigning the output of `runtime.block_on` to a variable, and then doing `match result {`